web_services/foxx/brightid/db.js

```diff
@@ -1039,6 +1046,11 @@ function isEthereumAddress(address) {
   return re.test(address);
 }
 
+function isEthereumSignature(sig) {
+  const re = new RegExp(/^[A-Fa-f0-9]{130}$/);
+  return re.test(sig);
+}
```

Comment:
I would assume that if the context is using soulbound signatures, the vast majority of contextids will be valid ethereum signatures. To avoid unnecessary processing, we could perform this check only if recovering the signer fails to help with debugging.